src/cc_catalog_airflow/dags/provider_api_scripts/walters_art_museum.py

```diff
@@ -0,0 +1,268 @@
+
+            if json_response_inpydict_form.get("NextPage") is False:
+                cond = False
+
+        if image_list is not None:
+            break
+
+    if num_tries == retries - 1 and image_list is None:
+        logger.warning("No more tries remaining. Returning Nonetypes.")
+        return None
+    else:
+        # total_images = _process_image_list(image_list)
+        # logger.info(f"Total Images obtained: {total_images}")
+        return image_list
+
+
+def _build_query_param(
+        class_param=None,
+        default_query_params=DEFAULT_QUERY_PARAMS,
+        apikey=API_KEY
+):
+    query_param = default_query_params.copy()
+    query_param.update(
+        {
+            'classification': class_param,
+            'apikey': apikey
+        }
+    )
+
+    return query_param
+
+
+def _extract_response_json(response_dict):
+    if response_dict is not None and response_dict.status_code == 200:
+        try:
+            json_response_inpydict_form = response_dict.json()
+        except Exception as error:
+            logger.warning(f"Could not get image data in JSON.\n{error}")
+            json_response_inpydict_form = None
+    else:
+        json_response_inpydict_form = None
+
+    return json_response_inpydict_form
+
+
+def _extract_image_list_from_json(json_response_inpydict_form):
+    if (
+            json_response_inpydict_form is None
+            or str(
+                json_response_inpydict_form.get('ReturnStatus')
+            ).lower() != 'true'
+            or json_response_inpydict_form.get('Items') is None
+            or len(json_response_inpydict_form.get('Items')) == 0
+    ):
+        image_list = None
+    else:
+        image_list = json_response_inpydict_form.get('Items')
+
+    return image_list
+
+
+def _process_image_list(image_list):
+    total_images = 0
+    if image_list is not None:
+        for img in image_list:
+            total_images = _process_image(img)
+
+    return total_images
+
+
+def _process_image(img):
+    logger.debug(f'Processing Image: {img}')
+
+    foreign_landing_url = _get_foreign_landing_url(img)
+    image_url = _get_image_url(img)
+    thumbnail_url = _get_thumbnail_url(img)
+    license_url = "https://creativecommons.org/publicdomain/zero/1.0/"
+    foreign_identifier = _get_foreign_id(img)
+    creator, creator_url = _get_creator_info(img)
+    title = _get_title(img)
+    meta_data = _get_image_meta_data(img)
+
+    return image_store.add_item(
+            foreign_landing_url=foreign_landing_url,
+            image_url=image_url,
+            thumbnail_url=thumbnail_url,
+            license_url=license_url,
+            foreign_identifier=foreign_identifier,
+            creator=creator,
+            creator_url=creator_url,
+            title=title,
+            meta_data=meta_data,
+    )
+
+
+def _get_foreign_landing_url(img):
+    foreign_landing_url = None
+    foreign_landing_url = img.get("ResourceURL")
+    return foreign_landing_url

```

Comment:
This function has precisely the same effect as `img.get("ResourceURL")`.  This is because `None` is the default return value for the `dict.get` method of python.  Thus, all of these 'get' functions should be removed, and replaced with the bare call to `dict.get` where each is called.